src/components/sbb-toggle/sbb-toggle.scss

```diff
@@ -35,6 +35,11 @@
   --sbb-toggle-selected-option-border-color: var(--sbb-color-graphite-default);
 }
 
+:host([disabled='false']) {
```

Comment:
does this ever happen?

Reply:
It doesn't happen in our examples, but it could happen if the consumer sets the `disabled` attribute to 'false' (string). With this rule it would actually evaluate as `false`, otherwise it would still be as it were `true`. It's up to us to decide whether to cover or not this case, what do you think?

Reply:
I think we should not support this case. We assume, that a boolean property should be set as defined HTML standards. @kyubisation can you give your decision here? ;)
We also don't support it anywhere else (So far I know)